src/actions/AddSiteActions.js

```diff
@@ -105,7 +107,9 @@ export const siteDetailCheckboxWasClicked = ({siteDetailCheckboxKey}) => {
 
 export const attemptToUploadSite = ({title, description, directions, nearestTown, accessibility, facilities, features, price, coordinate, siteImageData}, navigate) => {
     const {longitude, latitude} = coordinate;
-    const uniqueTitle = `${title}${longitude}${latitude}`;
+    const uniqueTitle = _(`${title}${longitude}${latitude}`)
+        .replace(/ /g, '')
+        .slice(0,30);
```

Comment:
I was worried that our titles might get too long in firebase and cause an error. Now to create a `uniqueTitle` we concatenate the title, long and lat, and remove whitespace, but slice it off after 30 characters.